package.json

```diff
@@ -695,6 +704,12 @@
         "title": "Show Class Documentation Preview",
         "enablement": "!isWeb",
         "icon": "$(open-preview)"
+      },
+      {
+        "category": "ObjectScript",
+        "command": "vscode-objectscript.exportCurrentFile",
+        "title": "Export Current File",
```

Comment:
Related to my comment above, but I'm finding it hard to come up with a title I'm happy with. Maybe "Export Server Version of Current File".